presto-memory/src/test/java/com/facebook/presto/plugin/memory/TestMemorySmoke.java

```diff
@@ -16,40 +16,36 @@
 import com.facebook.presto.metadata.QualifiedObjectName;
 import com.facebook.presto.testing.MaterializedResult;
 import com.facebook.presto.testing.MaterializedRow;
-import com.facebook.presto.testing.QueryRunner;
-import org.testng.annotations.BeforeTest;
+import com.facebook.presto.tests.AbstractTestQueryFramework;
+import org.intellij.lang.annotations.Language;
 import org.testng.annotations.Test;
 
 import java.sql.SQLException;
 import java.util.List;
 
 import static com.facebook.presto.plugin.memory.MemoryQueryRunner.CATALOG;
-import static com.facebook.presto.plugin.memory.MemoryQueryRunner.createQueryRunner;
 import static com.facebook.presto.testing.assertions.Assert.assertEquals;
 import static java.lang.String.format;
 import static org.testng.Assert.assertTrue;
 
 @Test(singleThreaded = true)
 public class TestMemorySmoke
+        extends AbstractTestQueryFramework
 {
-    private QueryRunner queryRunner;
-
-    @BeforeTest
-    public void setUp()
-            throws Exception
+    protected TestMemorySmoke()
     {
-        queryRunner = createQueryRunner();
+        super(MemoryQueryRunner::createQueryRunner);
     }
 
     @Test
     public void testCreateAndDropTable()
             throws SQLException
     {
         int tablesBeforeCreate = listMemoryTables().size();
-        queryRunner.execute("CREATE TABLE test AS SELECT * FROM tpch.tiny.nation");
+        computeActual("CREATE TABLE test AS SELECT * FROM tpch.tiny.nation");
```

Comment:
These DDL commands should use `assertUpdate` which does some additional validation for DDL/DML. `computeActual` is normally used for queries that return a result set. For the table creation, we verify the update count matches the number of rows in the source table.

```java
assertUpdate("CREATE TABLE test AS SELECT * FROM tpch.tiny.nation", "SELECT count(*) FROM tpch.tiny.nation");

assertUpdate("DROP TABLE test");